keps/973-workload-priority/README.md

```diff
@@ -330,7 +333,10 @@ This change should be covered by unit tests.
 
 #### Integration tests
 
-This change should be covered by integration tests.
+The following scenarios will be covered with integration tests where `WorkloadPriorityClass` is used:
+- Controller tests related to job interfaces such as job_controller, mpi_job_controller, etc
+- Integration tests for scheduler and webhook where the existing integration tests already cover `PriorityClass`
+- Integration tests for workloadPriorityClass_controller
```

Comment:
Will we actually need a reconciler for WorkloadPriorityClass?

Reply:
The WorkloadPriorityClass controller would only reconcile `WorkloadPriorityClass`.
Therefore, having a reconciler might be optional.
What do you think?

Reply:
But what does the reconciler do? I don't think we require any functionality.

Reply:
Yeah, I agree. Then I will not implement workloadPriorityClass controller.